README.md

```diff
@@ -4,6 +4,9 @@
 
 ![React App flowchart](blueprint/images/flowchart.png "React app flowchart")
 
+:::primary
+ **Note**: This blueprint uses React v16. Higher versions will require additional webpack configuration.
+
```

Comment:
Missing the closing `:::`

Reply:
Thanks, actually didn't know they required them haha.